dask/templates/additional-worker-deployment.yaml

```diff
@@ -0,0 +1,165 @@
+{{- range $worker := .Values.additional_worker_groups }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "dask.fullname" . }}-worker
+  labels:
+    app: {{ template "dask.name" . }}
+    heritage: {{ $.Release.Service | quote }}
```

Comment:
The `quote` is still there so I am not sure why its giving a string error

Reply:
I needed to approve the CI again. Until we get this merged I have to manually approve each CI run as you are a new contributor.